components/screens/Multiplayer.tsx

```diff
@@ -119,7 +148,8 @@ export default function Multiplayer(props: any) {
               { /* I can join the match if: */
                 item.host.uid !== user?.id && /* I'm not the host */
                 item.players.length < 4 && /* There is an empty space in [players] */
-                (item.players && !item.players.find(p => p.id === user?.id)) && /* I haven't already joined */
+                (item.players && !item.players.find(p => p.id === user?.id)) && /* I haven't already joined this match */
```

Comment:
Instead of checking if the username is already in the list of players, can we add a property to the player that checks if they already joined a match (like an `inMatch` boolean or something similar)?

Reply:
@franchizzle I'm adding this to the wiki for now, might be good to bring up during codeGoals